### PR TITLE
Notification/res array

### PR DESCRIPTION
**Pull request by devandapaige**
# Pull Request Template
## Description
Add a summary that address the following:
 - motivation/purpose for the feature 
`We needed to get our frontend talking with the backend, and this project uses React-Redux state management, so I focused first on our Reducer and Action.`

 - What were those changes
`I created separate Action and Reducer files to help manage the state. In our main Notifications component, I added a connect() function to the export. Once we have our chron too we can configure this component more.`

 - What issues were fixed
`Issues - Reducer and Action created for the state management.`

 - [Provide the link to the user story it is referencing in Trello board or other supporting document](https://trello.com/b/qUDG8j06/pt20-story-squad-team-b)

## Commits Checklist
- [x] commits have clear title names 
- [x] commit has a descriptive message of what has changed 
- [x] more small commits than less big commits 

## Code Checklist 
- [x] code is formatted appropriately 
- [x] code meets DRY standards
- [x] no dead code (check for logs and print statements)  
- [x] no TODOS instead necessary comments 
- [x] code follows Story Squad standards for:
     - Language 
     - Framework 
     - Libraries   

### PRs needs to be reviewed by at least 2 team members
 - TPL is the 3rd reviewer 
 - Lots of comments and suggestions, please! 

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

**Review by Bradleydion — APPROVED** [review]
This looks great. It should also help with some of the other tasks such as the one Ian an I are working on. Great job!!

**Review by Iandecisiv — APPROVED** [review]
Looks good to me.

File: src/state/reducers/notificationReducer.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { notifications } from '../actions/index';
```
Comment (kevinstonge) [inline src/state/reducers/notificationReducer.js:1]: here you import the action type string through the notifications object in ../actions/index - but in 'src/state/actions/notificationActions.js' you specify the notification string explicitly. I recommend that you import notifications the same way in the notificationActions.js file.

**Review by kevinstonge — APPROVED** [review]
aside from the suggestion I added, looks great!

File: src/state/reducers/notificationReducer.js (hunk 1 above)
Comment (devandapaige) [reply src/state/reducers/notificationReducer.js:1]: Good call! I'll update this within my next PR - thanks for catching that. 

Note to self - I'm going to keep this comment from Kevin unresolved until I edit this in my next PR.

**Pull request merged**